build.psm1

```diff
@@ -150,6 +150,42 @@ function Start-PSBuild {
         Stop-Process -Verbose
     }
 
+    # Generate version constant for $PSVersionTable
+    function Start-PowerShellVersionGen {
+        $powershellVersion = git --git-dir="$PSScriptRoot/.git" describe --abbrev=60 --long
+        $matchVersion = ([regex]::Match($powershellVersion, "^v(.+)-(\d+)-g(.+)")).Groups.Value
+        $null, $ps6MajorVersion, $ps6MinorVersion, $ps6PatchVersion, $ps6LabelVersion = ([regex]::Match($matchVersion[1], "^(\d+).(\d+).(\d+)-(.+)")).Groups.Value
+        $Global:formattedVersion = "$($matchVersion[1])"
+        if ($($matchVersion[1]) -ne "0") {
+            $Global:formattedVersion += "  Additional commits: $($matchVersion[2])"
+        }
+        $Global:formattedVersion += "  Commit Hash: $($matchVersion[3])"
```

Comment:
@JamesWTruher the telemetry you're receiving is from @iSazonov running a build of PowerShell with this (not yet committed to master) change.